packages/L10n/src/locales/en.yml

```diff
@@ -33,8 +33,9 @@ en:
       confirm: Save
       decline: Decline
     datePicker:
-      calendar_header_format: MMMM YYYY
```

Comment:
Is this the breaking change?

Reply:
Yes.

Note: every time when I introduce breaking change, I do commit with `!` in message and write some comment why this change considered as breaking.

Example:
![image](https://user-images.githubusercontent.com/30345591/82657874-5e1ef980-9c26-11ea-8772-bf43e6d0d930.png)